MyCryptoHQ/MyCrypto
# Refreshes balances when token is chosen in combobox

Closes #2322

### Description

When you are selecting an address from DeterministicWalletsModal, if balances are loaded and you choose a token from select, token balances are never loaded.

### Changes

* Refreshes addresses when a token is selected in `DeterministicWalletsModal`

### Steps to Test

1. Go to DeterministicWalletsModal (from Trezor, Ledger or Mnemonic)
2. Wait for balances to get loaded
3. Select a token from -Token-
4. Check that every balance is loaded after a while
